lib/virtfs/xfs/allocation_group.rb

```diff
@@ -1,12 +1,11 @@
 require 'binary_struct'
-require 'util/miq-uuid'
+require 'virt_disk/disk_uuid'
 require 'stringio'
 require 'memory_buffer'
-require 'fs/xfs/superblock'
 
 require 'rufus/lru'
 
-module XFS
+module VirtFS::XFS
   # ////////////////////////////////////////////////////////////////////////////
 
   AG_FREESPACE = BinaryStruct.new([
```

Comment:
Agreed.  Legacy from the original implementation of the filesystem, but regardless this makes more sense with the modifications.  Across all classes in the filesystem where appropriate.